exonum-java-binding/app/pom.xml

```diff
@@ -21,11 +21,11 @@
   <parent>
     <groupId>com.exonum.binding</groupId>
     <artifactId>exonum-java-binding-parent</artifactId>
-    <version>0.7.0-SNAPSHOT</version>
+    <version>0.8.0-SNAPSHOT</version>
   </parent>
 
   <artifactId>exonum-java-app</artifactId>
-  <version>0.7.0-SNAPSHOT</version>
+  <version>0.8.0-SNAPSHOT</version>
```

Comment:
JFYI: Found points of optimization (such as we use Maven 3.5+):
- child version can be inherited from the parent
- We can define version property in parent and also use it in children. 

So, by applying above refactoring we'll reduce versions boilerplate and our poms will be like following:

`ejb/pom.xml`:
```
  <groupId>com.exonum.binding</groupId>
  <artifactId>exonum-java-binding-parent</artifactId>
  <version>${ejb.version}</version>
  <packaging>pom</packaging>

  <properties>
    <ejb.version>0.7.0-SNAPSHOT</ejb.version>
  </properties>
```
`ejb/common/pom.xml`:
```
  <parent>
    <groupId>com.exonum.binding</groupId>
    <artifactId>exonum-java-binding-parent</artifactId>
    <version>${ejb.version}</version>
  </parent>

  <artifactId>exonum-java-binding-common</artifactId>
  <packaging>jar</packaging>
```

Also, see [discussion](https://stackoverflow.com/a/51969067/2706190)

Reply:
Looks interesting, we can get back to that later. Flatten also brings a small benefit: it removes all unrelated things from the POMs uploaded to the central.

tests_profile is way more annoying than updating the versions with a single command once in a release :upside_down_face:

Reply:
Should I raise a ticket for that task?

Reply:
:+1: yes, please